worker/src/RTC/RateCalculator.cpp

```diff
@@ -98,21 +85,21 @@ namespace RTC
 		if (this->newestItemIndex < 0 || this->oldestItemIndex < 0)
 			return;
 
-		uint64_t newoldestTime = nowMs - this->windowSizeMs;
+		uint64_t newOldestTime = nowMs - this->windowSizeMs;
 
 		// Oldest item already removed.
-		if (newoldestTime <= this->oldestItemStartTime)
+		if (newOldestTime < this->oldestItemStartTime)
 			return;
 
 		// A whole window size time has elapsed since last entry. Reset the buffer.
-		if (newoldestTime > this->newestItemStartTime)
+		if (newOldestTime > this->newestItemStartTime)
 		{
 			Reset();
 
 			return;
 		}
 
-		while (this->oldestItemStartTime < newoldestTime)
+		while (this->oldestItemStartTime <= newOldestTime)
```

Comment:
This is the real fix @dsdolzhenko?

Reply:
Can we have a tests that confirms it?

Reply:
> This is the fix @dsdolzhenko?

Yes, it is.

> Can we have a tests that confirms it?

Do you mean a kind of test that you could just run locally to confirm the issue? 

In that case you can checkout to `b927e4c23c3fc4383357e57e84d0351fa2f31e3c` commit where I'd added the assertion but haven't made the fix in `RateCalculator` yet and run existing tests. The `wrap` section of `[rtp][bitrate]` test will fail on that assertion.